Cargo.toml

```diff
@@ -28,3 +28,4 @@ rp2040-hal = { path = "./rp2040-hal" }
 
 [patch.crates-io]
 rp2040-hal = { path = "./rp2040-hal" }
+i2c-pio = { git = "https://github.com/ithinuel/i2c-pio-rs", branch = "fix-pio-exec-instruction" }
```

Comment:
I guess the plan is to remove this as soon as the branch is merged?

Reply:
Yes